CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [unreleased]
 
+- update to @sap/csn-interop-effective-specification v1.0.0
```

Comment:
```suggestion
- update to @sap/csn-interop-effective-specification v1.0.1
```

Just a reminder that we should release 1.0.1 bugfix first (https://github.com/SAP/csn-interop-specification/pull/52)